### PR TITLE
Order packages alphabetically

### PR DESCRIPTION
**Pull request by znxDomain**
This PR adds an OrderBy so that packages are listed alphabetically in the manifest returned to the client.

Without this the sort order in undefined, as returned by the filesystem. (It appears to be by folder modification date currently...)

### CODE REVIEW

File: SDSetupBackend/Controllers/v1.cs [hunk 1]
```diff
@@ -86,7 +86,7 @@ public class v1 : ControllerBase {
                     DeletingFileStream stream = (DeletingFileStream) ZipFromFilestreams(files.ToArray(), uuid);
 
                     Program.generatedZips[uuid] = stream;
-                    stream.Timeout(30000);
+                    stream.Timeout(180000);
```
Comment (noahc3) [inline SDSetupBackend/Controllers/v1.cs]: You included the changes from your other PR here

File: SDSetupBlazor/Pages/Consoles.cshtml [hunk 2]
```diff
@@ -249,7 +249,7 @@
 
 
 			HttpClient client = new HttpClient();
-			client.Timeout = TimeSpan.FromSeconds(120);
+			client.Timeout = TimeSpan.FromSeconds(200);
```
Comment (noahc3) [inline SDSetupBlazor/Pages/Consoles.cshtml]: And here

File: SDSetupBackend/Controllers/v1.cs (hunk 1 above)
Comment (znxDomain) [reply SDSetupBackend/Controllers/v1.cs]: I thought I rolled back/extracted this change from this PR.  I'll fix that.

File: SDSetupBlazor/Pages/Consoles.cshtml (hunk 2 above)
Comment (znxDomain) [reply SDSetupBlazor/Pages/Consoles.cshtml]: I thought I rolled back/extracted this change from this PR.  I'll fix that.

**Comment by znxDomain** [comment]
I know what happened here, github automatically tracks changes from the source branch, which in this case was my forks master.  I'll close and reopen a new PR.

**Pull request closed without merging**